### DISCUSSION TITLE
web2开发如何能找到智能合约开发的工作？

### DISCUSSION THREAD

**Discussion by RikiRiki8471** [AmazingAng/WTF-Solidity#508] [General]
本人是做Java开发的，学习solidity有一段时间了，wtf上的课程学完了solidity应用和Ethers.js入门，但是想自己写一个web3项目，做一个Dapp之类的感觉还无从下手，对于下一阶段的学习有点迷茫，要达到能够去工作的程度，还需要学习哪方面的知识呢？

**Comment by crazyzhaokk0** [comment]
一样,现在如何呢?
